### DISCUSSION TITLE
Invalid Workflow for triggers that aren't even defined

### DISCUSSION THREAD

**Discussion by kennethtran93** [community/community#26461] [Actions]
<p>Hi there,</p>
<p>I keep getting emails about failed workflow runs when I haven’t set any actions to run on issue_comments.  Unsure of it is a misconfiguration at my end or something on GitHub’s end.  The Event type filter tells me failed workflows on issue_comments.  This only happens when I set actions to run on local repo only.</p>
<p>Thanks for the assist.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/kennethtran93">@kennethtran93</a> ,</p>
<p>Looks you’re encountering the same <a href="https://github.com/octokit/rest.js/issues/1606" rel="nofollow noopener">issue</a> as linked here: <a href="https://github.com/devaradhanm/sample-js/actions/runs/39070595/workflow" rel="nofollow noopener">https://github.com/devaradhanm/sample-js/actions/runs/39070595/workflow</a></p>
<p>Please ask private support for help: <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a></p>
<p>Thanks.</p>
